FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1287,9 +1287,9 @@ namespace edm {
             try {
               ServiceRegistry::Operate operate(serviceToken_);
               if (looper_) {
-                auto& lp = *(status->lumiPrincipal());
+                auto& lp1 = *(status->lumiPrincipal());
```

Comment:
I think the original can be moved to after this statement since the original does not seem to be used until later.